libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -563,6 +568,50 @@ async def search(
             json=payload,
         )
 
+    async def get_versions(self, assistant_id: str) -> list[Assistant]:
+        """List all versions of an assistant.
+
+        Args:
+            assistant_id: The assistant ID to delete.
+
+        Returns:
+            list[Assistant]: A list of assistants.
+
+        Example Usage:
+
+            assistant_versions = await client.assistants.get_versions(
+                assistant_id="my_assistant_id"
+            )
+
+        """  # noqa: E501
+
+        return await self.http.get(f"/assistants/{assistant_id}/versions")
+
+    async def change_version(self, assistant_id: str, version: int) -> Assistant:
```

Comment:
maybe should be called set_latest or something?

Reply:
added to my todo, will make sure to change before merging

Reply:
might make sense to keep these consistent in the API as well. maybe `update_version` / `set_version`?